brevitas/proxy/parameter_quant.py

```diff
@@ -200,13 +210,26 @@ def _weight_quant_init_impl(bit_width: Optional[int],
             else:
                 bit_width_impl = bit_width_impl_override
 
-            if bit_width_impl_type == BitWidthImplType.PARAMETER or \
+            if bit_width_impl_type != BitWidthImplType.CONST or \
                     bit_width_impl_type == BitWidthImplType.CONST and \
-                    scaling_impl_type == ScalingImplType.PARAMETER_FROM_STATS:
+                    norm_impl_type == NormImplType.SAME_AS_SCALING and \
+                    scaling_impl_type != ScalingImplType.STATS:
                 tensor_clamp_impl = TensorClamp()
             else:
                 tensor_clamp_impl = TensorClampSte()
```

Comment:
I need to rethink whether this is shifting logic behind what is currently being added